splash/qtrender_lua.py

```diff
@@ -758,6 +758,36 @@ def callback(reply):
     def autoload_reset(self):
         self.tab.autoload_reset()
 
+    def _check_mouse_coordinantes(self, x, y):
+        try:
+            assert isinstance(x, int) or isinstance(x, float)
+            assert isinstance(y, int) or isinstance(y, float)
+        except AssertionError:
+            raise ScriptError({
+                "argument": "x, y",
```

Comment:
Could you please split this error in two separate errors for each argument? In all other cases "argument" contains a single argument which fails